erpnext/public/scss/workshop_cp.scss

```diff
@@ -16,16 +16,12 @@
 		z-index: 0;
 	}
 
-	td,
-	th {
-		border: 1px solid #E6E6E6;
-		padding: 4px;
-	}
-
-	tr:nth-child(even) {
-		background-color: #FFF;
-	}
-
+	.vehicle-table {
```

Comment:
We don't want to style just for vehicles table ... we want to style for all relevant tables ... use a better name